bigbluebutton-html5/private/config/settings.yml

```
@@ -24,7 +24,6 @@ public:
     # audio. Default value is true
     # Firefox users: if no output devices is shown, you may set the flag
     # "media.setsinkid.enabled" to make it work properly
-    # enableDynamicAudioDeviceSelection: true
     #
```

Recovering the removed line above

```suggestion
    # enableDynamicAudioDeviceSelection: true
    #
```